app/views/items/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>All Items</h1>
+
+<% @items.each do |item| %>
+  <section id="test-items-index-<%= item.id %>">
```

Comment:
This seems like bad form to have in your final HTML file sent to the development site. Why not use the standard `item-<%= item.id %>` for the ID?